vitess.io/index.md

```diff
@@ -1,6 +1,7 @@
 ---
 layout: home
 permalink: /
+title: Vitess - Database Clustering System for Horizontal Scaling of MySQL
```

Comment:
Combined with your other change, the diff for the generated title is going to look like:

-    <title>Vitess / Vitess - Database Clustering System for Horizontal Scaling of MySQL</title>
+    <title>Vitess - Database Clustering System for Horizontal Scaling of MySQL | Vitess</title>

That would repeat "Vitess" in there. But I don't think we cannot really prevent the "| Vitess" here?

In doubt, I'm okay with this :)

Reply:
@michael-berlin I just pushed a change to #3164 that should fix this issue. It only outputs `page.title` for the homepage